### PR TITLE
💄Moved newsletter subscription to the top for better visibility

### PR DESCRIPTION
**Pull request by NadeekaJK**
Context: 
As per the email from Adam with subject:  "🔥 URGENT: Tina homepage | Move location of newsletter subscribe controls", this code change will improve the visibility of the newsletter subscription by moving it from the Footer.tsx to CloudBanner.tsx. 
The UX changes checked by Betty.

Before:
image

After:
image

image

image

### General Contributing:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/tinacms/tinacms.org/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

### CODE REVIEW

File: components/layout/CloudBanner.tsx [hunk 1]
```diff
@@ -3,31 +3,42 @@ import Link from 'next/link'
...
+        >
             Star
-          </GitHubButton>
-        </div>
-      </div>
+        </GitHubButton>
+    </div>
+    <div className="banner-content w-auto flex justify-center">
+        <div className="mr-1 mt-1 justify-top"><p>Stay in touch 👉</p></div>
+        <span><EmailForm isFooter /></span>
```
Comment (andrewwaltosssw) [inline components/layout/CloudBanner.tsx:33]: Do we need a span wrapping this form element? It seems to flex as intended without it

File: components/layout/Footer.tsx [hunk 2]
```diff
@@ -292,10 +292,6 @@ export const Footer = ({}) => {
 
       {/* Bottom */}
       <div className="flex flex-col lg:flex-row w-full lg:items-center lg:justify-between bg-gradient-to-br from-orange-600 via-orange-800 to-orange-900 text-white px-6 py-8 lg:px-12 gap-6">
-        <div className="flex items-center gap-3 whitespace-nowrap">
-          <span>Stay in touch 👉</span>
-          <EmailForm isFooter />
-        </div>
         <div className="flex drop-shadow-sm flex-wrap justify-end gap-6">
```
Comment (andrewwaltosssw) [inline components/layout/Footer.tsx:295]: You should remove the unused EmailForm import at the top of the file

File: components/layout/CloudBanner.tsx [hunk 3]
```diff
@@ -3,31 +3,42 @@ import Link from 'next/link'
...
+    </div>
+</div>
```
Comment (andrewwaltosssw) [inline components/layout/CloudBanner.tsx:35]: Indenting on this page is out of alignment. Can you try to format the page to bring it back into alignment?

**Comment by andrewwaltosssw** [comment]
I feel that having the subscribe button appear after focusing the input field is a strange interaction and a bit jarring:

image
**Figure: No subscribe CTA on landing**

image
**Figure: Subscribe CTA visible after input field focused**

**Comment by andrewwaltosssw** [comment]
I feel that the responsiveness at certain breakpoints can be improved:

image
**Figure: ~800px viewport width showing overlapping elements and squished text**

image
**Figure: Mobile view has large portion of screen taken up by header**

File: components/layout/CloudBanner.tsx (hunk 1 above)
Comment (NadeekaJK) [reply components/layout/CloudBanner.tsx:33]: No, we don't need it. It looks like it was there in the original code and when I moved the component to the top, it came with the code chunk. Removed it.

File: components/layout/CloudBanner.tsx (hunk 3 above)
Comment (NadeekaJK) [reply components/layout/CloudBanner.tsx:35]: Done

**Comment by NadeekaJK** [comment]
> I feel that having the subscribe button appear after focusing the input field is a strange interaction and a bit jarring:
> 
> image **Figure: No subscribe CTA on landing**
> 
> image **Figure: Subscribe CTA visible after input field focused**

Point taken. However, given this was the existing functionality, and is out of the scope of this task, I would advise raising this as a separate UX suggestion.

**Comment by NadeekaJK** [comment]
> I feel that the responsiveness at certain breakpoints can be improved:
> 
> image **Figure: ~800px viewport width showing overlapping elements and squished text**
> 
> image **Figure: Mobile view has large portion of screen taken up by header**

Yes, I was aware of this issue and was talking to one of our UX engineers on the format in smaller screens. This is now fixed.  (See screenshots in PR description)

File: components/layout/Footer.tsx (hunk 2 above)
Comment (NadeekaJK) [reply components/layout/Footer.tsx]: Done

**Review by andrewwaltosssw — APPROVED** [review]
LGTM 🙌  🔥

**Review by andrewwaltosssw — APPROVED** [review]

**Review by JackDevAU — APPROVED** [review]
Approved ✅ it looks okay BUT, 

For the record, I don't think we should move the newsletter subscription input to the top of the page. It adds too much bloat.

**Pull request merged**